src/pages/auth/Auth.tsx

```diff
@@ -34,18 +29,31 @@ function Auth() {
   const [inPageOf, setInPageOf] = useState<"main" | "chat" | "game">("main");
   const [chatList, setChatList] = useState<ChatListType[]>([]);
   const [myChatList, setMyChatList] = useState<ChatListType[]>([]);
-  const [chatUserList, setChatUserList] = useState<ChatUserListType | null>(null);
-  const [roomId, setRoomId] = useState<number | undefined>(0);
+  const socket = getSocket();
 
-  useEffect(() => {
-    updateChatRoomList(setChatList);
-    updateMyChatRoomList(setMyChatList);
-  }, );
+  const chatRoomListener = (res: ChatListType[]) => {
+    let tmp: ChatListType[] = [];
```

Comment:
```suggestion
    const tmp: ChatListType[] = [];
```

배열이 js에서는 포인터 개념이라 주소값 기준 "상수"인거라 내용물 변하는건 영향을 주지 않아요
그래서 const 사용 가능합니다!

Reply:
포인터 확인요